src/pipelines.py

```diff
@@ -290,9 +306,9 @@ def mask_postprocessing(model, config, save_output=False):
              'unet_weighted': {'train': partial(unet_weighted, train_mode=True),
                                'inference': partial(unet_weighted, train_mode=False),
                                },
+             'unet_tta': {'inference': unet_tta,
+                          },
              'unet_padded': {'inference': unet_padded,
                              },
-             'unet_padded_tta': {'inference': unet_padded_tta,
-                                 },
```

Comment:
@jakubczakon I think unet_padded_tta pipeline is still going to be used. Right now it gives worse results, because scale problems, but I think for generalizing, especially if we will add scaling to TTA, padding will still improve our score.

Reply:
@taraspiotr you can still use it by setting 

```yaml
   loader_mode: crop_and_pad
```
in the `neptune.yaml`